exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -54,6 +54,23 @@ public void testConcatOnNonEmptyListOfLists() {
                 ListOps.concat(listOfLists));
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testConcatOnNonEmptyListOfNestedLists() {
+        List<List<List<Character>>> listOfNestedLists = Arrays.asList(
+                Arrays.asList(Collections.singletonList('1'),
+                Collections.singletonList('2')),
+                Arrays.asList(Collections.singletonList('3'),
+                Collections.emptyList()),
+                Arrays.asList(Arrays.asList('4', '5', '6'));
+
+        assertEquals(
+                Arrays.asList(Collections.singletonList('1'),
+                Collections.singletonList('2'), Collections.singletonList('3'),
+                Collections.emptyList(), Arrays.asList(4, 5, 6)),
```

Comment:
Shouldn't the `4, 5, 6` list also be characters? 🙂

Reply:
@FridaTveit Thanks for the valuable feedback. I will rectify it soon. Could you also suggest me an IDE for running the project? I am having issues with IntelliJ

Reply:
No problem 🙂 Hm, what issues are you having with IntelliJ? Feel free to open an issue on this track to describe your problems and we can try to help 🙂 I'm using IntelliJ and have found it to work well (though setting things up for this project can be a bit tricky so always feel free to ask for help!).

Apart fom IntelliJ I often use [Visual Studio Code](https://code.visualstudio.com/) but I haven't used it for this project or any Java and gradle based project so I'm not sure how good the support is for that kind of setup.

There's also [Eclipse](https://www.eclipse.org/ide/) which has quite good Java support. I haven't used it for this project and when I've used it before for other things I've found it to be a bit slow and prone to crashing unexpectedly. But the last time I used it was a few years ago so it might have improved since 🙂 

@sjwarner-bp @Smarticles101 do you have any suggestions? 🙂

Reply:
@FridaTveit @sjwarner-bp I had problems setting up the project, as such I could not locate any of the files in the right path. Hence I tried Atom. Are there any pointers as to how to set it up correctly. I have used IntelliJ and worked just fine for me in previous occassions.

Reply:
I think I figured it out for the moment, will update if I have any more issues. Thanks

Reply:
@FridaTveit could you tell me how to build the project in IntelliJ? This is to check if my code compiles correctly.

Reply:
We have docs for how to install Java and run everything in the console (https://github.com/exercism/java/blob/master/docs/INSTALLATION.md) but as far as I can tell we don't actually have docs describing how to setup the project in an IDE, sorry. But you should be able to open the project as a gradle project in IntelliJ, open the gradle window, expand the gradle tasks for `exercises`, expand the gradle tasks for the exercise you're working on, expand `Tasks`, expand `Build`, right click the `build` task and click `Run` (or `Debug` if you want to debug it). Let me know if that doesn't work or if any of it doesn't make sense 🙂

@sjwarner-bp maybe we should open an issue to add documentation for at least how to setup and run the project in IntelliJ since that's what's most people are using?

Reply:
That sounds sensible to me @FridaTveit :slightly_smiling_face: 

As you've said, it should hopefully be as simple as pulling in a Gradle project!

Reply:
I've opened #1479 - feel free to edit it if there are some issues :slightly_smiling_face: